airbyte-migration/src/test/java/io/airbyte/migrate/MigrationCurrentSchemaTest.java

```diff
@@ -61,19 +61,6 @@ void testConfigsOfLastMigrationMatchSource() {
         .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
   }
 
-  // get all of the "current" jobs (in other words the one airbyte-db). get all of the configs
-  // from the output schema of the last migration. make sure they match.
-  @Test
-  void testJobsOfLastMigrationMatchSource() {
```

Comment:
this will be tackled in https://github.com/airbytehq/airbyte/issues/5857

Reply:
I can work on this as soon as the Databricks connector is done. Should be at most a few hours work.

As long as no one is touching the file-based migration in the interim, we should be safe.

Reply:
Maybe add `@Ignored` to it for now?

Reply:
Will switch to adding @Disabled and leave a todo for you